package.json

```diff
@@ -154,6 +155,17 @@
       }
     }
   },
+  "exports": {
+    ".": "./index.js",
+    "./module": "./module.js",
+    "./api": "./api/index.js",
+    "./common": "./common/Tracker.js",
+    "./common/layers": "./common/layers/layer.js",
+    "./common/mixins": "./common/mixins/*.js",
+    "./mapbox": "./mapbox/index.js",
+    "./ol": "./ol/index.js",
+    "./ol/Map": "./ol/Map.js"
+  },
```

Comment:
I think I included all the export paths, but please verify if they are all there

Reply:
I don't know if we have to put every of them .
I think . , module, api, common,mapbox and ol are the most important, the rest isn't

```
".": "./index.js",
￼    "./module": "./module.js",
￼    "./api": "./api/index.js",
￼    "./common": "./common/*.js",
￼    "./mapbox": "./mapbox/index.js",
￼    "./ol": "./ol/index.js",
```

Reply:
@oterral Fixed. LGTM?

Reply:
yes